CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contribution guidelines
+
+## Branching
+
+Create your own branches from `master` and use a descriptive prefix:
+
+```sh
+# For bug fixes, ideally finishing with the ticket number.
+fix/<branch name>-<ticket number>
+# For features, as well finishing with the ticket number if there is one.
+feature/<branch name>-<ticket number>
+# For general non-functional improvements (eg. developer tooling, CI)
+chore/<branch name>-<ticket number>
+```
+
+When you're done with your work, create a pull request and find another teammate to review the changes.
+
+## Code reviews
+
+We do code reviews to ensure our work is of good quality, and to share knowledge. Reviews come in all shapes and sizes – from thorough point-by-point pull request reviews in GitLab to simple discussion of the proposed changes.
+
+### Why we review each other’s work
+
+*   Improve the overall quality of the work.
+*   Enforce and propagate our standards of quality, best practices, work methodologies, and coding styles.
+*   Share the responsibility and knowledge for any given piece of work.
+*   Learn from each other’s approaches and skillsets.
+
+> Want to know more? Go have a look at [Code Reviews: Just Do It](https://blog.codinghorror.com/code-reviews-just-do-it/) from Jeff Atwood.
+
+### Guidelines for code reviews
+
+When presenting code for review, or reviewing code, follow these guidelines:
+
+*   Try to make the review beneficial for both reviewee and reviewer.
+*   Give enough information to the reviewer so they know where to start.
+*   Think about the bigger picture.
+*   Concentrate on the intention rather than the syntax – stylistic issues should be taken care of by your linter.
+*   Reviews do not need to focus on the negative - if a piece of code is good, it is worth stating as well.
+
```

Comment:
These points are really good, but I feel like they need some explainers on _how_ to achieve them.

e.g. `Give enough information to the reviewer so they know where to start.` - How? What kind of information?

Reply:
I think it's important for those guidelines to be very high-level, so they are more likely to stay relevant over time, between projects, and for different types of tasks on the same project. Taking your example, the kind of information that would be useful will be very different between bug reports, features, docs, tooling, etc.